runtime/src/mem/jemalloc/mem-jemalloc.c

```diff
@@ -43,9 +43,11 @@
 #endif
 
 static struct shared_heap {
+  chpl_bool isFixed;
   void* base;
   size_t size;
   size_t cur_offset;
+  size_t page_size;  // valid iff !isFixed
```

Comment:
I don't see any reason to store this in the shared_heap struct. Seems like it could just be a static var in the `!isFixed` branch of the alloc routine